src/book/02-system/01-foundations/05-mathematics.mdx

```diff
@@ -275,3 +277,36 @@ For measurements on the field we use field space. For field coordinates the **x-
 - [Wolfram MathWorld](http://mathworld.wolfram.com/)
 
 - [Eigen Documentation](http://eigen.tuxfamily.org/dox/)
+
+# Interpolation
+
+Let's say we have a collection of data points. We want to find a function that contains all these points. We call this function an interpolant. Often we use a polynomial to describe this function.
```

Comment:
```suggestion
Given a set of data points, often we want to find a function that contains all these points, which we call an interpolant. Interpolants are commonly polynomials, because they can approximate complex curves with a few points, with relatively low computational costs. However, interpolant polynomials which are locally accurate for the given data can have oscillation issues at the edges of the interval which is being interpolated on - [Runge's phenomenon](https://en.wikipedia.org/wiki/Runge%27s_phenomenon).
```